routes/v1/auctionHouse.js

```diff
@@ -0,0 +1,43 @@
+//CREDIT: https://github.com/Senither/hypixel-skyblock-facade (Modified)
+const { makeRequest, wrap } = require('../../utils/request');
+
+module.exports = wrap(async function (req, res) {
+    const auctionsRes = (await makeRequest(res, `https://api.hypixel.net/skyblock/auctions`)).data.auctions;
```

Comment:
does it only use the first page or did I not see something

Reply:
Yeah, only first page